src/FakeItEasy/Configuration/BuildableCallRule.cs

```diff
@@ -158,6 +158,24 @@ public virtual void ApplyWherePredicate(Func<IFakeObjectCall, bool> predicate, A
 
         public abstract void UsePredicateToValidateArguments(Func<ArgumentCollection, bool> predicate);
 
+        /// <summary>
+        /// Clones the part of the rule that describes which call is being configured.
+        /// </summary>
+        /// <returns>The cloned rule.</returns>
+        public BuildableCallRule CloneCallSpecification()
+        {
+            var clone = this.CloneCallSpecificationCore();
+            clone.wherePredicates.AddRange(this.wherePredicates);
+            return clone;
+        }
+
+        /// <summary>
+        /// When overridden in a derived class, returns an new instance of the same type and takes care of copying the

```

Comment:
nit: "a new"

And possibly "takes care of copying" → "copies". Not that there's anything _wrong_ with what you have. I leave it to you.